docs/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.29.0
```

Comment:
Don't you want to publish it as a patch version?

Reply:
maybe. But I don't want to include it in 2.28.0 since it has already been tested via RC.

Reply:
It's not so critical since it has workaround

Reply:
what's the workaround? @neSpecc